composer.json

```
@@ -10,7 +10,7 @@
         }
     ],
     "conflict": {
-        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0",
+        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0 || 3.5.*",
```

```suggestion
        "doctrine/dbal": "2.9.* <2.9.3 || 3.3.0 || 3.5.0",
```

We should only add specific patch versions and never a whole minor version.

---

But didn't we also add open constraints like `> a` in the past? It doesn't look like it's going to get fixed on their side, so we will likely drop the rule in the future anyways (when we found a workaround).

---

No, we did not.

---

As doctrine now fixed this, using `3.5.0` as the conflict is correct IMO. https://github.com/doctrine/dbal/pull/5795

> > But didn't we also add open constraints like `> a` in the past?
>
> No, we did not.

We did that several times AFAIK, but only if we planned on removing the conflict again after fixing the compatibility ourselves...